src/main/java/com/fastmail/jstpierre/petmanager/consumers/Consumers.java

```diff
@@ -0,0 +1,20 @@
+package com.fastmail.jstpierre.petmanager.consumers;
+
+import com.fastmail.jstpierre.petmanager.entities.petowners.PetOwner;
+import com.fastmail.jstpierre.petmanager.entities.pets.Pet;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+public class Consumers {
```

Comment:
En Java, c'est une bonne pratique de mettre les classes utilitaires comme celle-ci `final` et d'y ajouter un constructeur protégé afin d'éviter qu'on l'instancie comme suit :
```suggestion
public final class Consumers {

    private Consumers() {
        // Constructeur protégé
    }
```

Reply:
Got it! J'ai fait de même pour Predicates